pykickstart/commands/autopart.py

```diff
@@ -654,3 +654,38 @@ def _getParser(self):
 class RHEL9_AutoPart(RHEL8_AutoPart):
     pass
 
+
+class F38_AutoPart(F29_AutoPart):
+    def __init__(self, writePriority=100, *args, **kwargs):
+        F29_AutoPart.__init__(self, writePriority=writePriority, *args, **kwargs)
+        self.hibernation = kwargs.get("hibernation", False)
+        self.swap = kwargs.get("swap", False)
+
+    def __str__(self):
+        retval = F29_AutoPart.__str__(self)
+        if not self.autopart:
+            return retval
+
+        if self.hibernation:
```

Comment:
You can check `if self.autopart and self.hibernation` and remove the condition above.